locale/po/ar_EG.po

```diff
@@ -6719,42 +6666,43 @@ msgstr ""
 msgid "Saved id [_1]"
 msgstr ""
 
-#: old/bin/am.pl:1234
+#: old/bin/am.pl:1245
 msgid "Saving [_1] [_2]"
 msgstr ""
 
 #: UI/oe-save-warn.html:12
 msgid "Saving over an existing document.  Continue?"
 msgstr ""
 
-#: old/bin/aa.pl:936 old/bin/aa.pl:968 old/bin/gl.pl:260 old/bin/ir.pl:530
-#: old/bin/ir.pl:944 old/bin/is.pl:1051 old/bin/is.pl:591 old/bin/oe.pl:675
-#: old/bin/oe.pl:862
+#: old/bin/aa.pl:946 old/bin/aa.pl:979 old/bin/gl.pl:255 old/bin/ir.pl:535
+#: old/bin/ir.pl:996 old/bin/is.pl:1106 old/bin/is.pl:602 old/bin/oe.pl:683
+#: old/bin/oe.pl:910
 msgid "Schedule"
 msgstr ""
 
-#: old/bin/printer.pl:173 UI/journal/journal_entry.html:353
+#: old/bin/printer.pl:174 UI/journal/journal_entry.html:354
 msgid "Scheduled"
 msgstr ""
 
-#: lib/LedgerSMB/Scripts/payment.pm:690 lib/LedgerSMB/Template/UI.pm:157
+#: lib/LedgerSMB/Scripts/payment.pm:691 lib/LedgerSMB/Template/UI.pm:157
 #: old/bin/printer.pl:129 UI/lib/utilities.html:44
 msgid "Screen"
 msgstr "الشاشة"
 
-#: UI/Reports/filters/batches.html:70 UI/Reports/filters/budget_search.html:76
+#: UI/asset/search_asset.html:188 UI/asset/search_class.html:66
+#: UI/business_units/filter.html:41 UI/Reports/filters/batches.html:58
+#: UI/Reports/filters/budget_search.html:76
 #: UI/Reports/filters/cogs_lines.html:35
 #: UI/Reports/filters/contact_search.html:140
 #: UI/Reports/filters/overpayments.html:60 UI/Reports/filters/payments.html:131
-#: UI/Reports/filters/reconciliation_search.html:97
+#: UI/Reports/filters/reconciliation_search.html:82
 #: UI/Reports/filters/search_goods.html:446
 #: UI/Reports/filters/search_partsgroups.html:17
 #: UI/Reports/filters/search_pricegroups.html:17
-#: UI/Reports/filters/timecards.html:48 UI/Reports/filters/unapproved.html:78
-#: UI/asset/search_asset.html:188 UI/asset/search_class.html:66
-#: UI/business_units/filter.html:41 sql/Pg-database.sql:2562
-#: sql/Pg-database.sql:2570 sql/Pg-database.sql:2583 sql/Pg-database.sql:2593
-#: sql/Pg-database.sql:2651 sql/Pg-database.sql:2698 sql/Pg-database.sql:2761
+#: UI/Reports/filters/timecards.html:48 UI/Reports/filters/unapproved.html:69
+#: sql/Pg-database.sql:2561 sql/Pg-database.sql:2569 sql/Pg-database.sql:2582
+#: sql/Pg-database.sql:2592 sql/Pg-database.sql:2650 sql/Pg-database.sql:2697
+#: sql/Pg-database.sql:2760
```

Comment:
This change looks incorrect, because `UI/business_units/filter.html` *definitely* still has the "Search" string. (Check https://github.com/ledgersmb/LedgerSMB/blob/a5f4bdaee7dcff9b469a2ae50492aa41e96f5590/UI/business_units/filter.html#L41 )